components/automate-chef-io/content/docs/SAML.md

```diff
@@ -0,0 +1,133 @@
++++
+title = "SAML"
+description = "SAML Configuration"
+date = 2018-05-11T09:27:09+00:00
+draft = false
+bref = ""
+toc = true
+[menu]
+  [menu.docs]
+    parent = "configuring_automate"
+    weight = 50
++++
+
+## Authentication via Existing Identity Management Systems
+
+Chef Automate can integrate with existing SAML services to authenticate users in Chef Automate, and thus use their existing group memberships to determine their Chef Automate permissions.
+
+Chef Automate supports using both local users and externally managed users from an external identity provider (IdP).
+Both _one_ LDAP service (or MSAD for simplified configuration of Active Directory setups) and _one_ SAML IdP can be used.
+You do not need to configure an external IdP if you simply want to create users and teams local to Chef Automate.
+See the [Users]({{< relref "users.md" >}}) documentation for additional information.
+
+Chef Automate uses [Dex](https://github.com/dexidp/dex) to support SAML integrations.
+To configure authentication for your Chef Automate installation, create a TOML file that contains the partial SAML configuration.
+Then run `chef-automate config patch </path/to/your-file.toml>` to deploy your change.
+
+{{% warning %}}
+You may only integrate one IdP using SAML and one IdP using LDAP at a time.
+Chef Automate does not support using _two_ SAML IdPs or _two_ LDAP services simultaneously.
+{{% /warning %}}
+
+If you need to change your configured IdP, you will need to replace
+your existing configuration by following these steps:
+
+1. Run `chef-automate config show config.toml`.
+2. Edit `config.toml` to replace the `dex.sys.connectors` section with the config values for your new identity provider.
+3. Run `chef-automate config set config.toml` to set your updated config.
+
+{{< info >}}
+Users who log in via SAML will have a session time of 24 hours before needing to log in again.
+Local, MSAD, and LDAP users will have their Chef Automate sessions refreshed while they maintain an active
+browsing session of the Chef Automate UI or until they log out directly.
+{{< /info >}}
+
+## SAML
```

Comment:
maybe `SAML Configuration Settings` for the heading text? or something that's not repeating the page name would be rad if possible